users/vieting/experiments/switchboard/hybrid/feat/baseline_args.py

```diff
@@ -221,11 +227,19 @@ def get_returnn_config(
                 network["features"]["subnetwork"]["specaug"] = specaug_layer_jingjing(in_layer=["conv_h_act"])
                 network["features"]["subnetwork"]["conv_h_split"]["from"] = "specaug"
                 network["source"] = {"class": "copy", "from": "features"}
+                prolog = get_funcs_jingjing()
             else:
-                network["source"] = specaug_layer_jingjing(in_layer=["features"])
-            prolog = get_funcs_jingjing()
+                if specaug_time_only:
```

Comment:
It should be possible to have time-only SpecAugment after the first layer as well.